server/config/config.go

```diff
@@ -305,6 +305,13 @@ type HTTPBasicAuthConfig struct {
 	Password string `json:"password" yaml:"password"`
 }
 
+// PackagingConfig holds configuration to build and retrieve Fleet packages
+type PackagingConfig struct {
+	// GlobalEnrollSecret is the enroll secret that will be used to enroll
+	// hosts in the global scope
+	GlobalEnrollSecret string `yaml:"global_enroll_secret"`
```

Comment:
I wonder if we should keep this more general for when we add teams support, maybe the originally proposed `EnrollSecretPool`?

Reply:
Or perhaps `EnrollSecrets`? This could be an array of objects with an optional field `team_id`.

Reply:
yeah, I like that. @lucasmrod what do you think?

Reply:
I'd keep this one as is, and when we introduce teams we add a separate config for it. (to keep them more explicitly separate)